FAQ/en.md

```diff
@@ -0,0 +1,73 @@
+# Frequently Asked Questions
+
+## General
+
+### What is Quaver?
+
+Quaver is a community-driven rhythm game, aiming to bring VSGRs to the next level. The game is open to players of all skill levels, casual and seasoned.
```

Comment:
Quaver is a community-driven and open source competitive rhythm game. It is inspired by a variety of classic rhythm games and plans to expand on them by being focused on real-time competitive play.

Reply:
applied